modules/ting_search/plugins/content_types/search_backends.inc

```diff
@@ -51,7 +51,7 @@ function ting_search_backend_engines_content_type_render($subtype, $conf, $panel
 
     // Create the option as a link.
     $txt = format_string('Search @backend', array('@backend' => $backend['title']));
-    $title = l(t('@txt', array('@txt' => $txt)), $path, array('query' => $conditions));
+    $title = l($txt, $path, array('query' => $conditions));
```

Comment:
Shouldn't this be:
$txt = t("Search @backend", array('@backend' => $backend['title']));
$title = l($txt, $path, array('query' => $conditions));

Reply:
You're right.. there's something wrong.

However, I changed it to:
$txt = format_string('Search @backend', array('@backend' => $backend['title']));
$title = l(t($txt), $path, array('query' => $conditions));

Doing it the suggested way, will prevent us from translating the backend['title'], which is returned untranslated from search_get_info(): https://api.drupal.org/api/drupal/modules%21search%21search.api.php/function/hook_search_info/7.x
We can only translate "Search @backend" in this case.

We could pass backend['title'] through a separate call to t(), but I suggest doing it this way, since it will work with the existing translations (it was done this way before).